pom.xml

```diff
@@ -50,11 +50,12 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <gradle.version>5.6</gradle.version>
+        <gradle.version>8.9</gradle.version>
         <gson.version>2.11.0</gson.version>
         <moshi.version>1.12.0</moshi.version>
         <jackson2x.version>2.17.2</jackson2x.version>
-        <jackson2x.databind.version>2.17.2</jackson2x.databind.version>
+        <ecj.version>3.42.0</ecj.version>
```

Comment:
It seems like we should declare the version we release with and then use profiles to support later JVMs.  Thoughs?

Reply:
I agree on the purpose.  It just feels like our releases are targeting JDK 8, so the main build with no profiles active should do that.  Then, we add support for these later JVMs as a convenience.  I often have JDK 17 running locally, so we are similarly motivated on this.